tests/wpunit/CheckoutMutationTest.php

```diff
@@ -85,181 +85,178 @@ function() {
+                                        id
+                                    }
+                                }
+                            }
+                            feeLines {
+                                nodes {
+                                    databaseId
+                                    orderId
+                                    amount
+                                    name
+                                    taxStatus
+                                    total
+                                    totalTax
+                                    taxClass
+                                }
+                            }
+                            shippingLines {
+                                nodes {
+                                    databaseId
+                                    orderId
+                                    methodTitle
+                                    total
+                                    totalTax
+                                    taxClass
+                                }
+                            }
+                            taxLines {
+                                nodes {
+                                    rateCode
+                                    label
+                                    taxTotal
+                                    shippingTaxTotal
+                                    isCompound
+                                    taxRate {
+                                        databaseId
+                                    }
+                                }
+                            }
+                            lineItems {
+                                nodes {
+                                    productId
+                                    variationId
+                                    quantity
+                                    taxClass
+                                    subtotal
+                                    subtotalTax
+                                    total
+                                    totalTax
+                                    taxStatus
+                                    product {
+                                        ... on SimpleProduct {
+                                            id
+                                        }
+                                        ... on VariableProduct {
+                                            id
+                                        }
+                                    }
+                                    variation {
+                                        id
+                                    }
+                                }
+                            }
+                        }
+                        customer {
+                            id
+                        }
+                        result
+                        redirect
+                    }
+                }
+            ';
+		}
```

Comment:
@scottyzen What's the purpose of this change? Also you didn't change the `checkout` mutation but the `createOrder` mutation. You should update the `OrderMutationsTest` instead.